pom.xml

```diff
@@ -103,6 +103,12 @@
 
 	<dependencies>
 		<dependency>
+      <groupId>org.awaitility</groupId>
```

Comment:
Formatting seems off. Also, I believe @artembilan used to have a problem with using this dependency, or is it OK for tests?

Reply:
Sounds good. I am not particular about awaitility; just needed some await library over using Thread.sleep.